.ci/onnx/test.sh

```diff
@@ -2,12 +2,16 @@
 
 # shellcheck source=./common.sh
 source "$(dirname "${BASH_SOURCE[0]}")/common.sh"
+source "$(dirname "${BASH_SOURCE[0]}")/../pytorch/common.sh"
```

Comment:
I'm not very familiar with `BASH_SOURCE`, do we need both lines?

Reply:
One comes from the current ONNX directory `.ci/onnx/common.sh`, while the other is from `.ci/pytorch/common.sh`.  The retry function is from the latter.  Despite their similar names, they have slightly different steps with some in common.  Let me see if I can combine them into one.  Having one less "common" file sounds nicer to me :)